core/src/test/java/org/apache/struts2/views/jsp/BeanTagTest.java

```diff
@@ -51,8 +51,8 @@ public void testSimple() {
             fail();
         }
 
-        request.verify();
-        pageContext.verify();
```

Comment:
Doesn't removing them skip some verifications that might fail with these changes? sorry I don't remember.

Reply:
@yasserzamani Thank you for your review.  I can say for sure that "pageContext.verify()" is a no-op.  As for the request, my belief is the mock is designed to let the tester set up a series of expectations, then call "verify" when the test is ready to check them.  However, this test seems to set no expectations, so my belief is the "verify" method here also would do nothing.

You touch here on what I consider to be the most risky of these changes:  The "mockobjects" library for the servlet api could no longer be used so I replaced them with the Spring library's mocks.  However, the Spring mocks do not support this "set/verify" pattern so in the few places our code uses it, I had to make a judgement as to what I thought the original intent was.  I cannot say for sure I got this right 100% of the time!

Reply:
Thanks for clarification @jdyer1 ! Yes you're right! I also couldn't find their usage. Those lines are old from 2006. Maybe underlying dependencies get updated making these left behind and why they don't make sense anymore.